### DISCUSSION TITLE
How to enable Full Path in Explorer View (using jeanp413 open-remote-ssh Extension)

### DISCUSSION THREAD

**Discussion by luckylinux** [VSCodium/vscodium#1954] [Q&A]
I tried to look into the Settings, but I can only see the latest Folder in my Path (`basename` basically).

E.g. for `/opt/myfolder` I can only see `myfolder` in Explorer ("UNTITLED (WORKSPACE)") and for `/srv` I would see `srv`.

Is there a way to configure VSCodium so that it would show Full Paths ?

And is there a way to give a Title to the Workspace ?

Thank you for your Help :+1:.

**Comment by Jeong-ilKajokaya** [comment]
Hi! Here’s how VSCodium/VSCode handles folder names and workspace titles:

---

### 1. Showing Full Paths in Explorer

By default, VSCode shows only the **last part of the folder** (basename) in the Explorer panel. There isn’t a native setting to show the **full path** directly in the Explorer.

**Workarounds:**

1. **Hover for full path:** Hover over the folder name in the Explorer — a tooltip shows the full path.
2. **Open multiple folders in a Workspace:** If you create a `.code-workspace` file and add multiple folders, you can assign custom names (see below). This also helps if you want to distinguish similarly named folders.

---

### 2. Setting a Title for the Workspace

You can assign a custom name to your workspace by creating a Workspace file:

1. `File -> Save Workspace As...`
2. Choose a name and location for the `.code-workspace` file.
3. Open the workspace file. You can edit it manually to assign friendly names to folders:

```json
{
  "folders": [
    {
      "path": "/opt/myfolder",
      "name": "My Opt Folder"
    },
    {
      "path": "/srv",
      "name": "Server Folder"
    }
  ],
  "settings": {}
}
```

* The `name` property will be shown in the Explorer instead of the basename.
* You can also set workspace-specific settings inside this file.

---

**Summary:**

* Full paths are not shown in Explorer by default; hover tooltips show full paths.
* Use a `.code-workspace` file and assign custom `name` properties to folders for easier identification.
* You can also name the workspace itself via `Save Workspace As...`.
